playbooks/roles/asqatasun/tasks/main.yml

```
@@ -0,0 +1,175 @@
+    name: postifx 
+    question: "{{ item.question }}" 
+    value: "{{ item.value }} " 
+    vtype: "string"
+  with_items:
+    - { question: "postfix/mailname", value: " " }
+    - { question: "postfix/main_mailer_type", value: "Satellite system" }
+  tags:
+    - install
+    - install:configuration
+
+- name: Update locale Setting
+  locale_gen:
+    name: "{{ locale }}"
+    state: present
+  register: set_locale
+  tags:
+    - install
+    - install:base
+
+- name: Reconfigure locale
+  command: dpkg-reconfigure locales
+  when: set_locale.changed
+
+- name: Install the Asqatasun Prerequisites
+  apt: 
+    name: "{{ item }}" 
+    update_cache: yes 
+    state: installed
+  with_items: asqatasun_debian_pkgs
+  tags:
+    - install
+    - install:base
+
+- name: Copy the asqatasun.cnf template to /etc/mysql/conf.d
+  template:
+    dest: /etc/mysql/conf.d/asqatasun.cnf
+    src: asqatasun.cnf.j2
+    owner: root
+    group: root
+  register: my_cnf
+  tags:
+    - install
+    - install:configuration
+
+- name: Restart MySQL
+  service: 
+    name: mysql 
+    state: restarted
+  when: my_cnf.changed
+
+- name: Create a soft link for tomcat jar and mysql connector
+  file:
+    dest: "{{ item.dest }}"
+    src: "{{ item.src }}"
+    state: link
+  with_items:
+    - { src: '/usr/share/java/spring3-instrument-tomcat.jar', dest: '/usr/share/tomcat7/lib/spring3-instrument-tomcat.jar' }
+    - { src: '/usr/share/java/mysql-connector-java.jar', dest: '/usr/share/tomcat7/lib/mysql-connector-java.jar'}
+  tags:
+    - install
+    - install:configuration
+
+- name: Copy the xvfb template to /etc/init.d
+  template:
+    dest: /etc/init.d/xvfb
+    src: xvfb.j2
+    owner: root
+    group: root
+    mode: 755
+  register: xvfb
+
+- name: Restart xvfb
+  service: 
+    name: xvfb 
+    pattern: /etc/init.d/xvfb 
+    state: restarted
+  when: xvfb.changed
+
+- name: Configure xvfb to run at startup
+  command: update-rc.d xvfb defaults
+  ignore_errors: yes
```

Why are we ignoring errors here?